<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quantics analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>17</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>17</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="DirectoryWidget" name="dir" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="add_flags_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Additional flags</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="add_flags">
         <property name="placeholderText">
          <string>Enter extra flags here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolBox" name="toolbox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="tabSpacing">
       <number>4</number>
      </property>
      <widget class="AnalysisConvergence" name="analconv">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>356</width>
         <height>292</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Analyse Convergence</string>
       </attribute>
      </widget>
      <widget class="AnalysisIntegrator" name="analint">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>356</width>
         <height>292</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Analyse Integrator</string>
       </attribute>
      </widget>
      <widget class="AnalysisResults" name="analres">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>356</width>
         <height>292</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Analyse Results</string>
       </attribute>
      </widget>
      <widget class="AnalysisSystem" name="analsys">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>356</width>
         <height>292</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Analyse System</string>
       </attribute>
      </widget>
      <widget class="AnalysisDirectDynamics" name="analdd">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>356</width>
         <height>292</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Analyse Direct Dynamics</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QTabWidget" name="tab_widget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="output_text_tab">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="CustomTextWidget" name="text">
          <property name="font">
           <font>
            <family>Courier</family>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>*********************************************************************************
                              QUANTICS ANALYSIS GUI
                                  version 1.1.0
*********************************************************************************

The Quantics analysis GUI allows convenient analysis of quantics output files
without the use of the CLI. Use the button in the top-right corner or Ctrl+O to
open a folder with output files.

Select an analysis command from the left toolbar, then press 'Continue' to see
the analysis. Switch between the text and plot widgets at the bottom of the
screen. When you are done with analysis, delete any leftover files by clicking
Options -&gt; Cleanup directory.

You can find additional options by right-clicking the text or plot widgets. For
more information, see the user guide in the Help menu.</string>
          </property>
          <property name="tabStopWidth">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="output_plot_tab">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="CustomPlotWidget" name="plot" native="true"/>
        </item>
        <item>
         <widget class="MediaWidget" name="media" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_dir"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_options">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menu_timeout">
     <property name="title">
      <string>Timeout</string>
     </property>
    </widget>
    <addaction name="menu_timeout"/>
    <addaction name="cleanup"/>
    <addaction name="allow_add_flags"/>
    <addaction name="no_command"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="open_guide"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_options"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_dir">
   <property name="text">
    <string>Open directory</string>
   </property>
   <property name="statusTip">
    <string>Opens a directory with quantics files.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="cleanup">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cleanup directory</string>
   </property>
   <property name="statusTip">
    <string>Remove all output files generated from quantics analysis programs.</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exits the program.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="no_command">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No command mode</string>
   </property>
   <property name="statusTip">
    <string>(Experimental) Do not execute quantics analysis commands, read from an existing analysis output file.</string>
   </property>
  </action>
  <action name="allow_add_flags">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Allow additional flags</string>
   </property>
   <property name="statusTip">
    <string>Shows a box where you can add additional flags to quantics analysis programs.</string>
   </property>
  </action>
  <action name="open_guide">
   <property name="text">
    <string>User guide</string>
   </property>
   <property name="statusTip">
    <string>Opens the user guide in your browser window.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlotWidget</class>
   <extends>QWidget</extends>
   <header>analysis_gui.ui.custom_plot</header>
  </customwidget>
  <customwidget>
   <class>AnalysisConvergence</class>
   <extends>QWidget</extends>
   <header>analysis_gui.analysis.convergence</header>
  </customwidget>
  <customwidget>
   <class>AnalysisIntegrator</class>
   <extends>QWidget</extends>
   <header>analysis_gui.analysis.integrator</header>
  </customwidget>
  <customwidget>
   <class>AnalysisResults</class>
   <extends>QWidget</extends>
   <header>analysis_gui.analysis.results</header>
  </customwidget>
  <customwidget>
   <class>AnalysisSystem</class>
   <extends>QWidget</extends>
   <header>analysis_gui.analysis.system</header>
  </customwidget>
  <customwidget>
   <class>AnalysisDirectDynamics</class>
   <extends>QWidget</extends>
   <header>analysis_gui.analysis.direct_dynamics</header>
  </customwidget>
  <customwidget>
   <class>CustomTextWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>analysis_gui.ui.custom_text</header>
  </customwidget>
  <customwidget>
   <class>DirectoryWidget</class>
   <extends>QWidget</extends>
   <header>analysis_gui.ui.dir_widget</header>
  </customwidget>
  <customwidget>
   <class>MediaWidget</class>
   <extends>QWidget</extends>
   <header>analysis_gui.ui.media_widget</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>text</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
